packages/did-core-test-server/suites/resolution-utils.js

```diff
@@ -0,0 +1,94 @@
+const findExpectedOutcome = ((expectedOutcomes, i) => {
+  const expectedOutcomesArray = [];
+  Object.keys(expectedOutcomes).forEach((expectedOutcome) => {
+        expectedOutcomes[expectedOutcome].forEach((expectedOutcomeIndex) => {
+          expectedOutcomesArray[expectedOutcomeIndex] = expectedOutcome;
+        });
+      });
+  return expectedOutcomesArray[i];
+});
+
+const isErrorExpectedOutcome = ((expectedOutcome) => {
+  return expectedOutcome.endsWith('ErrorOutcome');
+});
+
+const findExecutionByDid = ((implementation, did) => {
+  implementation.executions.forEach((execution) => {
+    if (execution.input.did == did) {
+      return execution;
+    }
+  });
+});
+
+const findExecutionByDidUrl = ((implementation, didUrl) => {
+  var found;
+  implementation.executions.forEach((execution) => {
+    if (execution.input.didUrl === didUrl) {
+      found = execution;
+      return;
+    }
+  });
+  return found;
+});
+
+const parseDidMethod = (did) => {
+  var match = /^did:(.+):(.+)$/.exec(did);
+  return match[1];
+};
+
+const produceRepresentation = (didDocument, contentType) => {
+  // TODO: improve this by re-using other test code
+  if (contentType.startsWith('application/did+json') ||
+      contentType.startsWith('application/did+ld+json')) {
+    return JSON.stringify(didDocument);
+  } else if (contentType.startsWith('application/did+cbor')) {
+    return '00';
+  }
+};
+
+const consumeRepresentation = (representation, contentType) => {
+  // TODO: improve this by re-using other test code
```

Comment:
nit: open an issue an link to it in todo... same could be said for the other todos.

Reply:
I agree, done: https://github.com/w3c/did-test-suite/issues/87